src/query/RelayFragmentPointer.js

```diff
@@ -66,11 +66,13 @@ class RelayFragmentPointer {
         });
       });
     }
+    let argType = typeof identifyingArgValue;
     invariant(
-      typeof identifyingArgValue === 'string' || identifyingArgValue == null,
+      typeof argType === 'string' || identifyingArgValue == null
```

Comment:
Since your aim is to support every type of argument, you should simply be able to remove this invariant.